docker-compose.yaml

```diff
@@ -77,15 +77,15 @@ services:
       - 8080:8080
       - 4000:4000
     volumes:
-      - ./developer/config/insecure-armada.yaml:/config/insecure-armada.yaml
+      - "./developer/config/insecure-armada.yaml:/config/insecure-armada.yaml"
       - "go-cache:/root/.cache/go-build:rw"
       - "gomod-cache:/go/pkg/mod:rw"
     depends_on:
       - eventingester
     working_dir: /app
     env_file:
       - ./developer/env/server-pulsar.env
-    command: ./server --config /config/insecure-armada.yaml
+    command: sh -c "sleep 30 && ./server --config /config/insecure-armada.yaml"
```

Comment:
We can surely find a better way of doing this, but for now I've just made this command mirror the one in `server-legacy` (the `sleep 30` part was added in #3047).